src/main/java/uk/gov/hmcts/sptribs/ciccase/model/CicCase.java

```diff
@@ -60,33 +57,6 @@
 @JsonNaming(PropertyNamingStrategies.UpperCamelCaseStrategy.class)
 public class CicCase {
 
-    @CCD(
-        label = "Enter case number",
-        access = {DefaultAccess.class, CaseworkerWithCAAAccess.class}
-    )
-    private CaseLink linkCaseNumber;
```

Comment:
As this field is being removed it will need to be retired and migrated

Reply:
yes Tom. Migration of old cases will be done as part of another ticket. The fields have to be retired and migrated.

Reply:
Fields removed from CicCase have been added to RetiredFields.